client/jetpack-onboarding/style.scss

```diff
@@ -13,11 +13,22 @@
 		}
 	}
 
+	.steps__summary {
+		.button {
+			position: relative;
+			left: 43%;
```

Comment:
Unfortunately, setting those types of positioning will not work consistently across devices. 

Also, once the button string is translated, the button width may be different, thus would require a different positioning.

So, none of this CSS is actually necessary, all we need is to center the button, and since it's an `inline-block`, all that's necessary is a `text-align: center` of the parent container. Good news: we already have a `className` for that. You can remove all of this CSS and simply wrap the button in a `<div className="steps__button-group" />`. This will center the button consistently across all devices and viewports.

Reply:
yes, true. Working on the follow-up I noted down that it is not quite right.